app/controllers/webhooks/mailchimp_unsubscribes_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Webhooks::MailchimpUnsubscribesController < ApplicationController
+  class InvalidListID < StandardError; end
+
+  LIST_MAPPINGS = {
+    mailchimp_newsletter_id: :email_newsletter,
+    mailchimp_sustaining_members_id: :email_membership_newsletter,
+    mailchimp_tag_moderators_id: :email_tag_mod_newsletter,
+    mailchimp_community_moderators_id: :email_community_mod_newsletter
+  }.freeze
+
+  def create
+    not_authorized unless valid_secret?
+    user = User.find_by!(email: params.dig(:data, :email))
```

Comment:
I decided to raise in case we can't find the user, but I'm open to other approaches (log to DataDog and no-op for example).

Reply:
This should be fine for the time being. It should be caught by honeybadger.

Reply:
I wonder if we should *not* raise while still testing this out/debugging, because if my assumption about the webhook payload are wrong we'll raise for no good reason.